<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SBMEECGControlPanelDlg</class>
 <widget class="QDialog" name="SBMEECGControlPanelDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ECG Control Panel</string>
  </property>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>292</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>01</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>02</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>03</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>04</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>05</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>06</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>07</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>08</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QCheckBox" name="checkBox_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QCheckBox" name="checkBox_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QCheckBox" name="checkBox_6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QCheckBox" name="checkBox_7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QCheckBox" name="checkBox_8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>09</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>11</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>12</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>13</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>14</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>15</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>16</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox_9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="checkBox_10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QCheckBox" name="checkBox_11">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QCheckBox" name="checkBox_12">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QCheckBox" name="checkBox_13">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QCheckBox" name="checkBox_14">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QCheckBox" name="checkBox_15">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QCheckBox" name="checkBox_16">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>17</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>18</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>19</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>21</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>22</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>23</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>24</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBox_17">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="checkBox_18">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QCheckBox" name="checkBox_19">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QCheckBox" name="checkBox_20">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QCheckBox" name="checkBox_21">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QCheckBox" name="checkBox_22">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QCheckBox" name="checkBox_23">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QCheckBox" name="checkBox_24">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>25</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>26</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>27</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>28</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>29</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>30</string>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string>31</string>
      </property>
     </widget>
    </item>
    <item row="6" column="7">
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string>32</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="checkBox_25">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="checkBox_26">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QCheckBox" name="checkBox_27">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QCheckBox" name="checkBox_28">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QCheckBox" name="checkBox_29">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QCheckBox" name="checkBox_30">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="6">
     <widget class="QCheckBox" name="checkBox_31">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="7">
     <widget class="QCheckBox" name="checkBox_32">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>33</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>34</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLabel" name="label_35">
      <property name="text">
       <string>35</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QLabel" name="label_36">
      <property name="text">
       <string>36</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QLabel" name="label_37">
      <property name="text">
       <string>37</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QLabel" name="label_38">
      <property name="text">
       <string>38</string>
      </property>
     </widget>
    </item>
    <item row="8" column="6">
     <widget class="QLabel" name="label_39">
      <property name="text">
       <string>39</string>
      </property>
     </widget>
    </item>
    <item row="8" column="7">
     <widget class="QLabel" name="label_40">
      <property name="text">
       <string>40</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QCheckBox" name="checkBox_33">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QCheckBox" name="checkBox_34">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QCheckBox" name="checkBox_35">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QCheckBox" name="checkBox_36">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QCheckBox" name="checkBox_37">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="5">
     <widget class="QCheckBox" name="checkBox_38">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="6">
     <widget class="QCheckBox" name="checkBox_39">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="7">
     <widget class="QCheckBox" name="checkBox_40">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_41">
      <property name="text">
       <string>41</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="label_42">
      <property name="text">
       <string>42</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QLabel" name="label_43">
      <property name="text">
       <string>43</string>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QLabel" name="label_44">
      <property name="text">
       <string>44</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QLabel" name="label_45">
      <property name="text">
       <string>45</string>
      </property>
     </widget>
    </item>
    <item row="10" column="5">
     <widget class="QLabel" name="label_46">
      <property name="text">
       <string>46</string>
      </property>
     </widget>
    </item>
    <item row="10" column="6">
     <widget class="QLabel" name="label_47">
      <property name="text">
       <string>47</string>
      </property>
     </widget>
    </item>
    <item row="10" column="7">
     <widget class="QLabel" name="label_48">
      <property name="text">
       <string>48</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QCheckBox" name="checkBox_41">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QCheckBox" name="checkBox_42">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QCheckBox" name="checkBox_43">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="3">
     <widget class="QCheckBox" name="checkBox_44">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="4">
     <widget class="QCheckBox" name="checkBox_45">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="5">
     <widget class="QCheckBox" name="checkBox_46">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="6">
     <widget class="QCheckBox" name="checkBox_47">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="7">
     <widget class="QCheckBox" name="checkBox_48">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_49">
      <property name="text">
       <string>49</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="label_50">
      <property name="text">
       <string>50</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QLabel" name="label_51">
      <property name="text">
       <string>51</string>
      </property>
     </widget>
    </item>
    <item row="12" column="3">
     <widget class="QLabel" name="label_52">
      <property name="text">
       <string>52</string>
      </property>
     </widget>
    </item>
    <item row="12" column="4">
     <widget class="QLabel" name="label_53">
      <property name="text">
       <string>53</string>
      </property>
     </widget>
    </item>
    <item row="12" column="5">
     <widget class="QLabel" name="label_54">
      <property name="text">
       <string>54</string>
      </property>
     </widget>
    </item>
    <item row="12" column="6">
     <widget class="QLabel" name="label_55">
      <property name="text">
       <string>55</string>
      </property>
     </widget>
    </item>
    <item row="12" column="7">
     <widget class="QLabel" name="label_56">
      <property name="text">
       <string>56</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QCheckBox" name="checkBox_49">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QCheckBox" name="checkBox_50">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <widget class="QCheckBox" name="checkBox_51">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="13" column="3">
     <widget class="QCheckBox" name="checkBox_52">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="13" column="4">
     <widget class="QCheckBox" name="checkBox_53">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="13" column="5">
     <widget class="QCheckBox" name="checkBox_54">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="13" column="6">
     <widget class="QCheckBox" name="checkBox_55">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="13" column="7">
     <widget class="QCheckBox" name="checkBox_56">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_57">
      <property name="text">
       <string>57</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLabel" name="label_58">
      <property name="text">
       <string>58</string>
      </property>
     </widget>
    </item>
    <item row="14" column="2">
     <widget class="QLabel" name="label_59">
      <property name="text">
       <string>59</string>
      </property>
     </widget>
    </item>
    <item row="14" column="3">
     <widget class="QLabel" name="label_60">
      <property name="text">
       <string>60</string>
      </property>
     </widget>
    </item>
    <item row="14" column="4">
     <widget class="QLabel" name="label_61">
      <property name="text">
       <string>61</string>
      </property>
     </widget>
    </item>
    <item row="14" column="5">
     <widget class="QLabel" name="label_62">
      <property name="text">
       <string>62</string>
      </property>
     </widget>
    </item>
    <item row="14" column="6">
     <widget class="QLabel" name="label_63">
      <property name="text">
       <string>63</string>
      </property>
     </widget>
    </item>
    <item row="14" column="7">
     <widget class="QLabel" name="label_64">
      <property name="text">
       <string>64</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QCheckBox" name="checkBox_57">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QCheckBox" name="checkBox_58">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="15" column="2">
     <widget class="QCheckBox" name="checkBox_59">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="15" column="3">
     <widget class="QCheckBox" name="checkBox_60">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="15" column="4">
     <widget class="QCheckBox" name="checkBox_61">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="15" column="5">
     <widget class="QCheckBox" name="checkBox_62">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="15" column="6">
     <widget class="QCheckBox" name="checkBox_63">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="15" column="7">
     <widget class="QCheckBox" name="checkBox_64">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
